packages/frontend/src/pages/instructlab/StartInstructLabContainer.svelte

```diff
@@ -0,0 +1,137 @@
+
+let { trackingId }: Props = $props();
+
+// The container provider connection to use
+let containerProviderConnection: ContainerProviderConnectionInfo | undefined = $state(undefined);
+
+// Filtered connections (started)
+let startedContainerProviderConnectionInfo: ContainerProviderConnectionInfo[] = $derived(
+  $containerProviderConnections.filter(connection => connection.status === 'started'),
+);
+
+// If the creation of the InstructLab fail
+let errorMsg: string | undefined = $state(undefined);
+// The containerId will be included in the tasks when the creation
+// process will be completed
+let containerId: string | undefined = $state(undefined);
+// available means the container is started
+let available: boolean = $derived(!!containerId);
+// loading state
+let loading = $derived(trackingId !== undefined && !errorMsg);
+
+onMount(async () => {
+  containerId = await instructlabClient.getInstructlabContainerId();
+});
+
+$effect(() => {
+  // Select default connection
+  if (!containerProviderConnection && startedContainerProviderConnectionInfo.length > 0) {
+    containerProviderConnection = startedContainerProviderConnectionInfo[0];
+  }
+});
+
+function processTasks(trackedTasks: Task[]): void {
+  // Check for errors
+  // hint: we do not need to display them as the TasksProgress component will
+  errorMsg = trackedTasks.find(task => task.error)?.error;
+
+  const task: Task | undefined = trackedTasks.find(task => 'containerId' in (task.labels ?? {}));
+  if (task === undefined) return;
+
+  containerId = task.labels?.['containerId'];
+}
+
+// Submit method when the form is valid
+async function submit(): Promise<void> {
+  errorMsg = undefined;
+  try {
+    trackingId = await instructlabClient.requestCreateInstructlabContainer({
+      connection: $state.snapshot(containerProviderConnection),
+    });
+    router.location.query.set('trackingId', trackingId);
+  } catch (err: unknown) {
+    console.error('Something wrong while trying to create the InstructLab container.', err);
+    errorMsg = String(err);
+  }
+}
+
+// Navigate to the new created service
+function openInstructLabContainer(): void {
+  instructlabClient.routeToInstructLabContainerTerminal(containerId!).catch(console.error);
+}
+</script>
+
+<FormPage title="Run InstructLab as a container">
+  <svelte:fragment slot="content">
+    <div class="flex flex-col w-full">
+      <header>
+        <div class="w-full flex flex-row px-8">
+          {#if available}
+            <Button inProgress={!available} title="Open InstructLab container" on:click={openInstructLabContainer}>
+              Open InstructLab container
+            </Button>
+          {:else}
+            <Button title="Start InstructLab container" inProgress={loading} on:click={submit}>
+              Start InstructLab container
+            </Button>
+          {/if}
+          <Button title="Read documentation" type="link" on:click={submit}>Read documentation</Button>
+        </div>
+      </header>
```

Comment:
![image](https://github.com/user-attachments/assets/5f84533d-4e80-417b-ac39-bd191d241d34)

A bit weird to have the buttons on the top ?

Reply:
When the action is not started, the buttons are at the top. 
Once the user clicked on the "Start" button, the triggered actions display bellow the action, which seems the natural reading order and avoid the page's primary actions to be moving in the screen.

Reply:
This is not really consistent with the other forms where the buttons are on the bottom right

Reply:
But that's not a form. There is only one action the user can do on this screen.